prow/config/README.md

```diff
@@ -78,6 +78,9 @@ requirement_presets:
       preset-service-account: "true"
 ```
 
+In each sub-folder, a `.base.yaml` file can also be added which'll overwrite the
```

Comment:
Need to clarify a bit more. Is it an overlay on top of the root base yaml, or does it completely replace it? Also, does this work recursively? For example:

```
./base.yaml (1)
./dir1/base.yaml (2)
./dir1/dir2/base.yaml (3)
```

For files under `dir1/dir2`, is the base just `dir1/dir2/base.yaml`? Or is it `./base.yaml` overlayed by `./dir1/base.yaml` overlayed by `./dir1/dir2/base.yaml`?

Reply:
It is an overlay on top of the root base yaml. Currently the overlay logic only applies to the `.base.yaml` files in the current folder and root folder since that'll work for us for now, but in the future I may change it to work recursively.
I've added more docs for clarification.